spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb

```diff
@@ -108,6 +113,9 @@
       end
 
       it "will perform a full refresh" do
+        # NOTE: for backporting compatibility, where tagging does not implemented for the Graph refresh
```

Comment:
The comment does not sound English ;-)
I am not sure how to fix it. Please, ask some native speaker.

Reply:
Not sure of the context but I'd suggest either "is not implemented" or "can not be implemented".

Reply:
Perhaps "where tag mapping was not implemented"

Reply:
lets just fix this and then I think its ready

Reply:
done